kube-monitoring/charts/values.yaml

```
@@ -1,14 +1,13 @@
 global:
-
-  ## Labels to apply to all resources
-  ##
+  # -- Labels to apply to all resources
+  # This can be used to add a `support_group` or `service` label to all resources and alerting rules.
+  # @section -- Global Options
```

To stay consistent with other section headers. 
```suggestion
  # @section -- Global options
```